content-src/components/NewTabPage/NewTabPage.js

```diff
@@ -6,12 +6,19 @@ const {GroupedActivityFeed} = require("components/ActivityFeed/ActivityFeed");
 const Spotlight = require("components/Spotlight/Spotlight");
 const Search = require("components/Search/Search");
 const {actions} = require("actions/action-manager");
+let REPORT_RENDER_EVENT = true;
 
 const NewTabPage = React.createClass({
   // TODO: Replace with real search api via addon
   onSearch(value) {
     this.props.dispatch(actions.NotifyPerformSearch(value));
   },
+  componentWillUpdate(nextProps, nextState) {
+    if (nextProps.isReady && REPORT_RENDER_EVENT) {
+      this.props.dispatch(actions.NotifyTelemetry("NEWTAB_RENDER"));
```

Comment:
@k88hudson  could you comment on @oyiptong question.  Here's the relevant code snippet in NewTabPage.js:
```
  componentWillUpdate(nextProps, nextState) {
    if (nextProps.isReady && REPORT_RENDER_EVENT) {
      this.props.dispatch(actions.NotifyTelemetry("NEWTAB_RENDER"));
      REPORT_RENDER_EVENT = false;
    }
```
I vaguely recall that the reason was not to send another NEWTAB_RENDER events to perfMeter if the page is re-rendered due to a history event.  Could you verify that this theory is correct.